.github/workflows/appstore-build-publish.yml

```diff
@@ -136,24 +136,24 @@ jobs:
       - name: Sign app
         run: |
           # Extracting release
-          cd ${{ env.APP_NAME }}/build/artifacts/appstore
+          cd ${{ env.APP_NAME }}/build/artifacts
           tar -xvf ${{ env.APP_NAME }}.tar.gz
-          cd ../../../../
+          cd ../../../
           # Setting up keys
           echo "${{ secrets.APP_PRIVATE_KEY }}" > ${{ env.APP_NAME }}.key
           wget --quiet "https://github.com/nextcloud/app-certificate-requests/raw/master/${{ env.APP_NAME }}/${{ env.APP_NAME }}.crt"
           # Signing
-          php nextcloud/occ integrity:sign-app --privateKey=../${{ env.APP_NAME }}.key --certificate=../${{ env.APP_NAME }}.crt --path=../${{ env.APP_NAME }}/build/artifacts/appstore/${{ env.APP_NAME }}
+          php nextcloud/occ integrity:sign-app --privateKey=../${{ env.APP_NAME }}.key --certificate=../${{ env.APP_NAME }}.crt --path=../${{ env.APP_NAME }}/build/artifacts/${{ env.APP_NAME }}
           # Rebuilding archive
-          cd ${{ env.APP_NAME }}/build/artifacts/appstore
```

Comment:
Mh, I would have liked to have a say in merging this. This breaks the publishing step.